pkg/lintcontext/create_contexts.go

```diff
@@ -115,6 +115,8 @@ func CreateContextsWithOptions(options Options, filesOrDirs ...string) ([]LintCo
 }
 
 // CreateContextsFromHelmArchive creates a context from TGZ reader of Helm Chart.
+// Note: although this function is not used in CLI, it is exposed from kube-linter library and therefore should stay.
+// See https://github.com/stackrox/kube-linter/pull/173
```

Comment:
This generally applies to all functions we have that are not in the `internal` directory. Not sure if it's really required to call it out.

Reply:
I'd keep it for my future self or someone who'll try to do it like me. If that's ok with you